specification/operationalinsights/data-plane/readme.typescript.md

```diff
@@ -6,7 +6,7 @@ Please also specify `--typescript-sdks-folder=<path to the root directory of you
 ``` yaml $(typescript)
 typescript:
   package-name: "@azure/loganalytics"
-  output-folder: "$(typescript-sdks-folder)/packages/@azure/loganalytics"
+  output-folder: "$(typescript-sdks-folder)/sdk/loganalytics/loganalytics"
```

Comment:
We should try to find out if the service name is loganalytics or operationalinsights/

Reply:
For now I have renamed it to `operationalinsights`